content/docs/use-cases/ml-model-registries.md

```diff
@@ -0,0 +1,54 @@
+# Machine Learning Model Registries
+
+An **ML model registry** (or _model catalog_) works as a central repository
+between data science projects and their applications. ML models and other data
+artifacts can be browsed, shared, audited, tested, and deployed from here. DVC
+repositories support all of this while integrating seamlessly with existing
+development workflows. Other tools from [Iterative](https://iterative.ai/)
+further improve your model catalogs with more features and better usability.
+
+![](/img/ml_model_registry_placement.jpg) _Model registry placement contrasts
+with general [data registries] for reusable data sources or training sets._
+
+<!-- TODO: Look up package registry diagrams for inspiration -->
+
+[data registries]: /doc/use-cases/data-registries
+
+DVC model registries provide your team with key collaborative capabilities:
```

Comment:
can we emphasize basic things - it's Git-based, etc ....

Reply:
in the list or in the paragraph above

Reply:
I added that DVC projects are Git-based. I think most other DVC basics are already covered but I'll double check... ⌛